applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -109,6 +110,16 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
         if options.nrf21540:
             device_flag += " -DSHIELD=nrf21540ek_fwd"
 
+        if options.user_bt_name:
+            MAX_USER_NAME_LEN = 248
+            user_specific_bt_name = ("AUDIO_DEV_" + getpass.getuser())[:MAX_USER_NAME_LEN].upper()
+            device_flag += " -DCONFIG_BT_DEVICE_NAME=\\\"" + user_specific_bt_name + "\\\""
+
+        if options.custom_bt_name is not None:
+            MAX_USER_NAME_LEN = 248
```

Comment:
I suggest putting an "if !options.user_bt_name:" check here and only use custom_bt_name if user_bt_name is not set